code/modules/library/lib_machines.dm

```diff
@@ -433,12 +437,6 @@ var/global/list/datum/cachedbook/cachedbooks // List of our cached book datums
 				B.icon_state = "book[rand(1,7)]"
 				src.visible_message("[src]'s printer hums as it produces a completely bound book. How did it do that?")
 				break
-	if(href_list["orderbyid"])
-		var/orderid = input("Enter your order:") as num|null
-		if(orderid)
-			if(isnum(orderid))
-				var/nhref = "src=\ref[src];targetid=[orderid]"
-				spawn() src.Topic(nhref, params2list(nhref), src)

```

Comment:
It's pre-everything.

This code is new enough that goon coders never worked on it, but old enough that babby tgcoders worked on it before the knew what maintainability is.